cloudpickle/cloudpickle.py

```diff
@@ -238,7 +238,11 @@ def dump(self, obj):
 
     def save_memoryview(self, obj):
         """Fallback to save_string"""
-        Pickler.save_string(self, str(obj))
+        try:
+            arg = obj.obj
+        except ValueError:
+            arg = _released_memory_view_sentinel
+        self.save_reduce(_fill_memoryview, (arg,), obj=obj)
```

Comment:
How do we serialize `v[1:]` without it? Right now `v[1:]` would become `v`because we can't get the offset info.